### PR TITLE
Tpwdcee 4824

### PR DESCRIPTION
**Pull request by brkicadis**

### CODE REVIEW

File: .bin/get-shopware-versions.sh [hunk 1]
```diff
@@ -7,4 +7,4 @@ git config --global user.email "wirecard@travis-ci.org"
 
 git add  ${SHOPWARE_RELEASES_FILE}
 git commit -m "[skip ci] Update latest shop releases"
-git push --quiet https://${GITHUB_TOKEN}@github.com/${TRAVIS_REPO_SLUG} HEAD:master
+git push --quiet https://${GITHUB_TOKEN}@github.com/${TRAVIS_REPO_SLUG} HEAD:TPWDCEE-4824
```
Comment (flavius) [inline .bin/get-shopware-versions.sh]: Why this change? Looks big.

Comment (brkicadis) [reply .bin/get-shopware-versions.sh]: done
